libs/openFrameworks/graphics/ofTrueTypeFont.cpp

```diff
@@ -371,9 +371,11 @@ static string linuxFontPathByName(const string & fontname){
 
 //-----------------------------------------------------------
 // FIXME: it makes no sense to have _fontname and filename if filename will be rewritten inside this function
-static bool loadFontFace(const of::filesystem::path& _fontname, FT_Face & face, of::filesystem::path & filename, int index){
+// FIXME: it seems first parameter is string because it represents the font name only
+static bool loadFontFace(const of::filesystem::path & _fontname, FT_Face & face, of::filesystem::path & filename, int index){
 	auto fontname = _fontname;
-	filename = ofToDataPathFS(_fontname,true);
+	// FIXME: ALERT : review this, it seems font name and filename are confused here.
+	filename = ofToDataPathFS(_fontname, true);
```

Comment:
this is correct - as _fontname can be a file path or a font name. 
if the file path is not found it is treated as a font name.